frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -116,7 +123,18 @@ const PVCTableRow = ({ obj, index, key, style }) => {
       <TableData className={tableColumnClasses[4]}>
         {_.get(obj, 'status.capacity.storage', '-')}
       </TableData>
-      <TableData className={tableColumnClasses[5]}>
+      <TableData className={classNames(tableColumnClasses[5])}>
+        {obj?.spec?.storageClassName ? (
+          <ResourceLink
+            kind="StorageClass"
+            name={obj?.spec?.storageClassName}
+            title={obj?.spec?.storageClassName}
+          />
+        ) : (
+          <div className="text-muted">-</div>
```

Comment:
Most places we don't use `text-muted` for `-`

Reply:
@spadgett Will address this  and the removal of Bootstrap grid on a new PR.